OpenRA.Mods.Common/Traits/BotModules/SquadManagerBotModule.cs

```diff
@@ -126,6 +126,21 @@ public bool IsEnemyUnit(Actor a)
 				&& !a.GetEnabledTargetTypes().IsEmpty;
 		}
 
+		public bool IsNotHiddenUnit(Actor a)
+		{
+			var hasModifier = false;
+			var visModifiers = a.TraitsImplementing<IVisibilityModifier>().ToArray();
```

Comment:
Now this `ToArray` is one too many again, sorry.

Reply:
It is OK here. I also update my back up.